### DISCUSSION TITLE
How do I use a bytes wav object instead of passing a referential string for a path to a wav file?

### DISCUSSION THREAD

**Discussion by feliperab** [snakers4/silero-models#198] [Q&A]
Hello, consider this code:

```
def transcribe(file):
    torch.hub.download_url_to_file('https://opus-codec.org/static/examples/samples/speech_orig.wav',
                                dst ='speech_orig.wav', progress=True)
    test_files = glob('speech_orig.wav')
    batches = split_into_batches(test_files, batch_size=10)
    input = prepare_model_input(read_batch(batches[0]),
                                device=device)
    output = model(input)
    return decoder(output[0])
```

I want to use "file" (a bytes object) instead of "speech_orig.wav", how do I do it?

If I replace it in any of the steps I get errors.

**Comment by snakers4** [comment]
You can just override this function - https://github.com/snakers4/silero-models/blob/20ce121e0deb7a1ce8fa007f27efdfff6e3dd816/src/silero/utils.py#L22-L37

**Reply by feliperab** [reply]
> You can just override this function -
> 
> https://github.com/snakers4/silero-models/blob/20ce121e0deb7a1ce8fa007f27efdfff6e3dd816/src/silero/utils.py#L22-L37

I am sorry about asking this, but can you provide an example of how it can be done? Do I need to edit that file directly?
